modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the form view
+
+You can use a similar procedure to customize the following UI items:
+** Quick starts visible to users
```

Comment:
This should be a level 1 unordered list marked with `*`, what you currently have is a level 2 list: `**`.

Reply:
Fixed.